jibri: correct chromedriver mismatch

benbz opened the pull request:
Previously when `CHROME_RELEASE` is set to something specific (not `latest`), the `[ "${CHROMEDRIVER_MAJOR_RELEASE}" = "latest" ]` check is skipped due to the `||` before it. i.e. the original code is structured like:

```sh
CHROME_RELEASE=95.0.4638.69 CHROMEDRIVER_MAJOR_RELEASE=95 echo "Do initial things" && \
  [ "${CHROME_RELEASE}" = "latest" ] && \
  echo "Do latest chrome release things 1" && \
  echo "Do latest chrome release things 2"  || \
  [ "${CHROME_RELEASE}" != "latest" ] && \
  echo "Do specific chrome release 1" && \
  echo "Do specific chrome release 2" || \
  [ "${CHROMEDRIVER_MAJOR_RELEASE}" = "latest" ] && \
  CHROMEDRIVER_RELEASE="$(curl -4Ls https://chromedriver.storage.googleapis.com/LATEST_RELEASE)" || \
  CHROMEDRIVER_RELEASE="$(curl -4Ls https://chromedriver.storage.googleapis.com/LATEST_RELEASE_${CHROMEDRIVER_MAJOR_RELEASE})" && \
  echo "${CHROMEDRIVER_MAJOR_RELEASE} vs ${CHROMEDRIVER_RELEASE}"

Do initial things
Do specific chrome release 1
Do specific chrome release 2
95 vs 96.0.4664.45
```

The new code on the other hand:
```sh
CHROME_RELEASE=95.0.4638.69 CHROMEDRIVER_MAJOR_RELEASE=95 echo "Do initial things" && \
  [ "${CHROME_RELEASE}" = "latest" ] && \
  ( \
    echo "Do latest chrome release things 1" && \
    echo "Do latest chrome release things 2" \
  ) || \
  ( \
    echo "Do specific chrome release 1" && \
    echo "Do specific chrome release 2" \
  ) && \
  [ "${CHROMEDRIVER_MAJOR_RELEASE}" = "latest" ] && \
  CHROMEDRIVER_RELEASE="$(curl -4Ls https://chromedriver.storage.googleapis.com/LATEST_RELEASE)" || \
  CHROMEDRIVER_RELEASE="$(curl -4Ls https://chromedriver.storage.googleapis.com/LATEST_RELEASE_${CHROMEDRIVER_MAJOR_RELEASE})" && \
  echo "${CHROMEDRIVER_MAJOR_RELEASE} vs ${CHROMEDRIVER_RELEASE}"

Do initial things
Do specific chrome release 1
Do specific chrome release 2
95 vs 95.0.4638.69
```

This PR fixes this, makes things easier to debug in future and adds a fail-safe incase there's another issue

saghul commented:
Thanks! This image is already huge, so I think having an extra layer where we run a script (let's say some install-chrome.sh we copy) would be best for maintainability.

benbz commented:
> Thanks! This image is already huge, so I think having an extra layer where we run a script (let's say some install-chrome.sh we copy) would be best for maintainability.

Ack, sounds good to me. I'll try to update this PR with that this afternoon

saghul commented:
Awesome, thanks a ton!

benbz commented:
> > Thanks! This image is already huge, so I think having an extra layer where we run a script (let's say some install-chrome.sh we copy) would be best for maintainability.
> 
> Ack, sounds good to me. I'll try to update this PR with that this afternoon

Done in 4afeb5a:
* `set -e` in the script so we don't need to chain everything with `&&`
* Chrome & Chromedriver are installed in a separate layer so that changing their versions doesn't invalid the main apt layer. I'm happy to merge it all back down to a single layer if preferred
* I moved the installation of `jitsi-upload-integrations` & `jq` into the main `apt` run rather than a 3rd apt run at the end. I couldn't see a reason why they needed to be installed after Chrome but again happy to change this back if there's something I've missed
* Fixed an issue where the gnome GPG key wasn't being saved correctly when `CHROME_RELEASE=latest`
* Removed the check I added in 62a7154, it didn't work when `CHROMEDRIVER_MAJOR_RELEASE=latest` and then felt a bit pointless

lephat08 commented:
Hi, I have issues with Jibri. I deployed Jibri on docker. However, when I started recording, I got problems. Could you help me how to resolve it? :(
![image](https://user-images.githubusercontent.com/45971257/208586817-cc82f05e-db5d-411c-8802-cfbc6cc95d01.png)
